packages/salesforcedx-vscode-apex-debugger/package.json

```diff
@@ -15,7 +15,101 @@
   "engines": {
     "vscode": "^1.13.0"
   },
-  "categories": [
-    "Debuggers"
-  ]
+  "categories": ["Debuggers"],
+  "dependencies": {
+    "@salesforce/salesforcedx-utils-vscode": "40.4.0",
+    "vscode-debugprotocol": "1.19.0",
+    "vscode-debugadapter": "1.19.0"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.0.0",
+    "@types/mocha": "2.2.38",
+    "@types/node": "^6.0.40",
+    "@types/sinon": "1.16.31",
+    "chai": "^4.0.2",
+    "mocha": "3.2.0",
+    "sinon": "1.17.4",
+    "nyc": "^11.0.2",
+    "typescript": "2.4.0",
+    "vscode": "1.1.2",
+    "vscode-debugadapter-testsupport": "1.19.0",
+    "mock-spawn": "0.2.6"
+  },
+  "scripts": {
+    "vscode:prepublish": "npm prune --production",
+    "vscode:package": "vsce package",
+    "vscode:sha256": "node ../../scripts/generate-sha256.js >> ../../SHA256",
+    "vscode:publish": "node ../../scripts/publish-vsix.js",
```

Comment:
You probably want to remove this so that when I run the weekly publish, this will not be published to the VS Code Marketplace yet.